src/Workspaces/Core/Portable/Storage/SQLite/v1/SQLitePersistentStorage.cs

```diff
@@ -241,14 +242,14 @@ private void CloseWorker()
         private PooledConnection GetPooledConnection()
             => new PooledConnection(this, GetConnection());
 
-        public void Initialize(Solution solution)
+        public void Initialize(Solution solutionOpt)
```

Comment:
tried to NRT this file, but it was a bit too much.  so leaving that for the future.